filament/backend/src/opengl/platforms/PlatformCocoaTouchGL.mm

```diff
@@ -88,6 +88,9 @@
             pImpl->mGLContext, nullptr, &pImpl->mTextureCache);
     assert_invariant(success == kCVReturnSuccess);
 
+    // Necessary to avoid unused-variable warning in release build.
+    (void)success;
```

Comment:
you can use `UTILS_UNUSED_IN_RELEASE` instead. e.g.:

```
UTILS_UNUSED_IN_RELEASE bool success;
```

Reply:
done